runtask.yml

```diff
@@ -12,13 +12,19 @@ environment:
         - pdc-client
         - python2-aexpect
         - python2-avocado
+        - python2-avocado-plugins-varianter-yaml-to-mux
```

Comment:
Be sure to install the corresponding `python2-avocado` from the updates-testing repo as well.